Nitrox.Launcher/ViewModels/OptionsViewModel.cs

```diff
@@ -53,6 +53,12 @@ internal partial class OptionsViewModel(IKeyValueStore keyValueStore, StorageSer
     [ObservableProperty]
     private bool allowMultipleGameInstances;
     
+    [ObservableProperty]
+    private bool steamOverlayEnabled;
+    
```

Comment:
Same thing as my other comment, I'm not seeing where this variable is used. But I'm reviewing on my phone so maybe I missed something.